aten/src/ATen/native/native_functions.yaml

```diff
@@ -8966,6 +8959,26 @@
   dispatch:
     Math: linalg_tensorsolve_out
 
+- func: linalg_qr(Tensor self, str mode='reduced') -> (Tensor Q, Tensor R)
+  python_module: linalg
+  use_c10_dispatcher: full
+  variants: function
+  dispatch:
+    DefaultBackend: linalg_qr
+
+- func: linalg_qr.Q(Tensor self, str mode='reduced', *, Tensor(a!) Q, Tensor(b!) R) -> (Tensor(a!) Q, Tensor(b!) R)
```

Comment:
Same here, I would expect this one to be `linalg_qr.out` as it is an out variant.